config/features.yml

```diff
@@ -407,11 +407,6 @@ features:
     enable_in_development: true
     description: >
       Manage dependent removal from view dependent page
-  vanotify_service_enhancement:
```

Comment:
I have never _removed_ a feature toggle before - is there any issue with removing it here but still having entries for this toggle in the database? Guessing not but something to verify looks right in staging I guess.